### ISSUE TITLE
canShareVia('whatsapp') says no, but shareViaWhatsApp works?

### ISSUE THREAD

**Issue by arielweinberger** [EddyVerbruggen/SocialSharing-PhoneGap-Plugin#575]
Hello.
I am running SocialSharing on iOS 9.
So I have this function:
```
    $cordovaSocialSharing.canShareVia(
        'whatsapp',
        'message',
        null,
        null,
        null
      )
      .then(function (res) {
        console.log('Can share via WhatsApp');
      }, function (err) {
        console.log('Cannot share via WhatsApp');
      });
```
This one executes the error callback, meaning I cannot share via whatsapp.

But if I run this:
`$cordovaSocialSharing.shareViaWhatsApp('message');`

I am able to share via WhatsApp. Is this a bug?

**Comment by arielweinberger** [comment]
Solved by adding 'whatsapp' and 'instagram' to the `LSApplicationQuriesSchemes` array in info.plist.

Still wondering, is there any way to add values to this array via Cordova's config.xml?

**Comment by erperejildo** [comment]
I'm facing the same issue. Where is `LSApplicationQuriesSchemes ` to add that info?

**Comment by natronite** [comment]
To add entries to `LSApplicationQueriesSchemes` in the plist file use the cordova-custom-config plugin. According to the [documentation](https://github.com/dpa99c/cordova-custom-config#ios-project-plist-config-blocks) you would add whatsapp and instagram like so:

```
<platform name="ios">
   ...
   <config-file parent="LSApplicationQueriesSchemes" target="*-Info.plist">
      <array>
         <string>whatsapp</string>
         <string>instagram</string>
      </array>
   </config-file>
</platform>
```

**Comment by sohanpc** [comment]
Yes i got same issue on android canShareVia('whatsapp') says no, but shareViaWhatsApp works . any solution ?
